src/routes/blog/post/announcing-database-reads-and-writes-pricing/+page.markdoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Announcing Database reads and writes pricing
```

Comment:
I think we should use the chicago style for titles - https://capitalizemytitle.com/style/chicago/